tests/cpu/test_kalman_fitter_wire_chamber.cpp

```diff
@@ -193,7 +193,7 @@ INSTANTIATE_TEST_SUITE_P(
     ::testing::Values(std::make_tuple(
         "wire_10_GeV_muon", std::array<scalar, 3u>{0.f, 0.f, 0.f},
         std::array<scalar, 3u>{0.f, 0.f, 0.f},
-        std::array<scalar, 2u>{10.f, 10.f}, std::array<scalar, 2u>{-1.f, 1.f},
+        std::array<scalar, 2u>{10.f, 10.f}, std::array<scalar, 2u>{-0.3f, 0.3f},
```

Comment:
@krasznaa Just commenting on this change, the eta range is changing from [-1,1] -> [-0.3, 0.3].

Yeah so nothing has been fixed yet and I am avoiding to face the problem in this PR